File: Project 1-10/project2.py
#ProjectEuler Number 2
#by Josh Smith

sum = 0
fib = 0
x = 0
y = 1
while fib < 4000000:
    fib = x + y
    x = y
    y = fib
    if(fib%2 == 0):
	sum = sum + fib

print sum

File: C++/project2.cpp
//ProjectEuler Number 2
//by Josh Smith

#include <iostream>
using namespace std;

int main() {
	int fTotal = 0;
	int fib;
	int x = 0;
	int y = 1;
	while(fib < 4000000){
		fib = x + y;
		x = y;
		y = fib;
		if(fib%2 == 0)
			fTotal += fib;
	}
	cout << fTotal << endl;
    
    return 0;
}
